app/products/calls/screens/host_controls/host_controls.tsx

```diff
@@ -43,19 +43,40 @@ export const HostControls = ({
     const intl = useIntl();
     const {bottom} = useSafeAreaInsets();
 
+    const sharingScreen = currentCall.screenOn === session.sessionId;
+
     const makeHostPress = useCallback(async () => {
-        await makeHost(currentCall.serverUrl, currentCall.channelId, session.userId);
+        await hostMake(currentCall.serverUrl, currentCall.channelId, session.userId);
```

Comment:
Do we want to wait for the action and then dismiss the bottom sheet? Normally is the other way around, we dismiss the sheet and then perform the action cause if it fails the sheet will remain opened and there is no feedback anyway, causing a bit of confusion imho

Reply:
Yep, makes sense, done. Thanks